tests/test_filtsmooth/test_statespace/test_integrator.py

```diff
@@ -30,7 +30,7 @@ def test_proj2coord(self):
 
 
 STEP = np.random.rand()
-DIFFCONST = np.random.rand()
+DIFFCONST = np.random.rand() ** 2
```

Comment:
"diffusion" means sigma^2 now

Reply:
Thanks for the comment. Lets be careful with the fact that this sentiment should be reflected everywhere.